tests/integration/test_lists/qa/llm_sanity_test.txt

```diff
@@ -136,9 +136,12 @@ accuracy/test_llm_api_pytorch.py::TestMixtral8x7B::test_nvfp4_tp2
 accuracy/test_llm_api_pytorch.py::TestNemotronNas::test_auto_dtype_tp8
 accuracy/test_llm_api_pytorch.py::TestNemotronSuper::test_auto_dtype_tp2
 accuracy/test_llm_api_pytorch.py::TestNemotronNano::test_auto_dtype
+accuracy/test_llm_api_pytorch.py::TestNemotronUltra::test_auto_dtype[tp8-cuda_graph=False]
+accuracy/test_llm_api_pytorch.py::TestNemotronUltra::test_fp8_tp8[tp8-cuda_graph=False]
```

Comment:
Super nit:  can we rename this test to `test_fp8_<QUANT_ALG>` instead of `..._tp8`, since tp8 is already reflected in the following parameterized ID?